reflex/reflex.py

```diff
@@ -78,6 +78,11 @@ def run(
     frontend_port: str = typer.Option(None, help="Specify a different frontend port."),
     backend_port: str = typer.Option(None, help="Specify a different backend port."),
     backend_host: str = typer.Option(None, help="Specify the backend host."),
+    db_migrate: bool = typer.Option(
```

Comment:
Since we have the `db` subcommand now, it may be cleaner to only have one way of migrating and not support auto migrate in the `run` command anymore.

Reply:
Yeah, I think that might be a simpler mental model:

Initialize like before on blank database, otherwise print the error message and tell the user to explicitly migrate.

Reply:
updated in the latest commit